cognite/client/data_classes/iam.py

```diff
@@ -140,7 +141,7 @@ def is_managed_in_cdf(self) -> bool:
         return self.members is not None
 
     @classmethod
-    def _load(cls, resource: dict, cognite_client: CogniteClient | None = None, allow_unknown: bool = False) -> Group:
+    def _load(cls, resource: dict, cognite_client: CogniteClient | None = None, allow_unknown: bool = True) -> Group:
```

Comment:
I really don't think we should change this default to true

Reply:
If CDF has a group with a capability which is not implemented in the ACL, when doing retrieve/list then we will fail to load the response from the API. 

My thinking is that in the read format (Group/GroupList) we should accept whatever capabilities the API returns.

Reply:
This is hardcoded into the `group.list` and `token.inspet` methods today, but is not there for `.create`

Reply:
Right, yeah, agree that create should be updated to match 👌 Good catch!